src/status_im/qr_scanner/core.cljs

```diff
@@ -1,6 +1,8 @@
 (ns status-im.qr-scanner.core
   (:require [re-frame.core :as re-frame]
             [status-im.i18n :as i18n]
+            [status-im.ui.screens.navigation :as navigation]
+            [taoensso.timbre :as log]
```

Comment:
Import not used?

Reply:
Oh yeah, I was using logs to debug. I'll remove this.